pkg/skaffold/build/build.go

```diff
@@ -28,6 +28,8 @@ import (
 type Artifact struct {
 	ImageName string `json:"imageName"`
 	Tag       string `json:"tag"`
+	// Config is the  corresponding artifact object from `skaffold.yaml`.
+	Config latest.Artifact `json:"-"`
```

Comment:
I'm not sure I understand where this gets populated from?

Reply:
Hrmm that seems to have somehow disappeared.  And clearly a test is needed.